### PR TITLE
Average calculation

### PR DESCRIPTION
**Pull request by Saikiruthiga**
Average calculation program using command line arguments

### CODE REVIEW

File: nodejs/week1/avg.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+for (let i = 2; i < process.argv.length; i++) {
+  let num = Number(process.argv[i]);
+  if (isNaN(num)) {
+    console.log(`All the arguments should be a number`);
+    return;
+  } else {
+    total += num;
+    count++;
+  }
+}
```
Comment (magdazelena) [inline nodejs/week1/avg.js]: this looks great, how about trying to do the same using array functions? :)

**Review by magdazelena — APPROVED** [review]

File: nodejs/week1/avg.js (hunk 1 above)
Comment (Saikiruthiga) [reply nodejs/week1/avg.js]: Thank you for your review. Array function used.

**Review by babak-f — APPROVED** [review]
Good job :)

File: nodejs/week1/avg.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// Program for calculate the average of the numbers provided as command line arguments.
+
+const num = process.argv.slice(2).map(Number);
+
+if (num.some(isNaN)) {
+  console.log("The arguments should be a number");
+  return;
+}
+if (num.length > 0) {
+  const total = num.reduce((acc, curr) => acc + curr, 0);
```
Comment (magdazelena) [inline nodejs/week1/avg.js:10]: 😍

**Pull request merged**